### DISCUSSION TITLE
How to identify differentially unregulated protein markers defining a cluster?

### DISCUSSION THREAD

**Discussion by denvercal1234GitHub** [ImmuneDynamics/Spectre#159] [Q&A]
Hi there, 

Thanks again for the tool. 

Would you mind giving me some advice regarding tools or packages to identify which markers define a cluster? in scRNAseq transcriptomic analysis, there is DEG analysis. Is there something similar for flow data?

Thank you for your help!

**Comment by ghar1821** [comment]
@denvercal1234GitHub have a look at diffcyt (https://www.nature.com/articles/s42003-019-0415-5), which essentially employs edgeR and limma (classically used in transcriptomics) to find differentially expressed proteins. Just bear in mind that it can be cumbersome to use if you are not familiar with design matrix.

**Reply by denvercal1234GitHub** [reply]
Thank you @ghar1821! You guys have very well-explained and detailed tutorials for other things on the website. Do you have plan to do one for this analysis, which I think is quite useful?

**Reply by ghar1821** [reply]
We can certainly put it on the card. What do you think @tomashhurst ?

**Reply by denvercal1234GitHub** [reply]
That would be very helpful! Thanks @ghar1821 and @tomashhurst

**Comment by tomashhurst** [comment]
@denvercal1234GitHub @ghar1821 definitely on the cards, setting up a flow from Spectre --> edgeR has been on my mind for a while, and will definitely feature in the v2 updates. However, in the meantime, we can probably spin up something quickly -- in this case it would really just be getting the data.table into the right format.

**Reply by denvercal1234GitHub** [reply]
Hi @tomashhurst @ghar1821 - I wonder whether you guys might have had a chance to look into this differential protein expression analysis for Spectre workflow? Thank you for your help!
